caravel/templates/caravel/export_dashboards.html

```diff
@@ -0,0 +1,6 @@
+<script>
+    window.onload = function() {
+        window.open(window.location += '&action=go');
+        window.location = '{{ dashboards_url }}';
```

Comment:
the challenge here is to download a file and redirect right after, flask doesn't provide an easy way to do that :(